src/test/robot/lib/MongooseContainer.robot

```diff
@@ -1,31 +1,55 @@
 *** Settings ***
 Documentation  Mongoose Container Keywords
-Library  OperatingSystem
-Library  RequestsLibrary
+Resource       Common.robot
+Library        OperatingSystem
+Library        RequestsLibrary
+
+
 
 *** Variables ***
-${LOG_DIR} =  build/log
-${MONGOOSE_CONTAINER_DATA_DIR} =  /data
-${MONGOOSE_CONTAINER_NAME} =  mongoose
-${MONGOOSE_IMAGE_NAME} =  emcmongoose/mongoose-base
-${MONGOOSE_NODE_PORT} =  9999
+${LOG_DIR}                      build/log
+${MONGOOSE_CONTAINER_DATA_DIR}  /data
+${MONGOOSE_CONTAINER_NAME}      mongoose
+${MONGOOSE_IMAGE_NAME}          emcmongoose/mongoose-base
+
+
 
 *** Keywords ***
+Start Mongoose Nodes
+    Start Entry Mongoose Node
+    Start Additional Mongoose Node
+
+Start Entry Mongoose Node
+    Start Mongoose Node  ${SESSION_NAME}  ${MONGOOSE_REST_PORT}  ${MONGOOSE_RMI_PORT}
+
+Start Additional Mongoose Node
+    Start Mongoose Node  ${ADD_SESSION_NAME}  ${MONGOOSE_ADD_REST_PORT}  ${MONGOOSE_ADD_RMI_PORT}
+
+Remove Mongoose Nodes
+    Delete All Sessions
+    Run  docker stop ${SESSION_NAME}
+    Run  docker rm ${SESSION_NAME}
+    Run  docker stop ${ADD_SESSION_NAME}
+    Run  docker rm ${ADD_SESSION_NAME}
+
 Start Mongoose Node
+    [Arguments]  ${session_name}  ${rest_port}  ${rmi_port}
     ${image_version} =  Get Environment Variable  MONGOOSE_IMAGE_VERSION
     # ${service_host} should be used instead of the "localhost" in GL CI
     ${service_host} =  Get Environment Variable  SERVICE_HOST
     ${cmd} =  Catenate  SEPARATOR= \\\n\t
     ...  docker run
     ...  --detach
-    ...  --name ${MONGOOSE_CONTAINER_NAME}
-    ...  --publish ${MONGOOSE_NODE_PORT}:${MONGOOSE_NODE_PORT}
+    ...  --name ${session_name}
+    ...  --publish ${rest_port}:${MONGOOSE_REST_PORT}
+    ...  --publish ${rmi_port}:${MONGOOSE_RMI_PORT}
     ...  ${MONGOOSE_IMAGE_NAME}:${image_version}
-    ...  --load-step-id=robotest
-    ...  --run-node
+    ...  --load-step-id=${STEP_ID} --run-node
```

Comment:
is there any specific reason for these two to be on the same line?

Reply:
No, It is typo